plugins/lookup/conjur_variable.py

```diff
@@ -31,6 +31,10 @@
         description: Flag to control SSL certificate validation
         type: boolean
         default: True
+      file_mode:
+        description: Store lookup result in a temp file
```

Comment:
This should include another sentence regarding what this is useful for.

Reply:
I agree, done in next commit.